.github/workflows/pages-deploy.yml

```diff
@@ -0,0 +1,49 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets the GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  # Single deploy job since we're just deploying
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
```

Comment:
Do you think we can update this to be 23 and also update this line over in the other action?
https://github.com/CodeCafeCommunity/codecafecommunity.github.io/blob/f1821a016811d2197cd683b1020998ecca4853a3/.github/workflows/format-and-lint.yml#L14

I'd like us to use the latest version where we can.
I will also open up another ticket to have a `.nvmrc` or `.node-version` file that specifies version 23

Reply:
Actually, I took care of the one in `format-and-lint.yml` in #13, so no need to change that file.